### PR TITLE
Pioneer MAB deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates for CSM and glider deployments.  The CSM deploy csv updates have been reviewed by Sawyer already before I posted them to the PR.  But glider deploy csvs need 2 reviews.

QLR tables and updated configs attached.
Glider sensor SNs can be verified in the FCPs (3304-00150-00168 --> -00170)

### CODE REVIEW

File: deployment/CP10CNSM_Deploy.csv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+AR82,,,,CP10CNSM-MFD36-04-VELPTA000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-VELPTA-08544,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD36-05-PCO2WB000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-PCO2WB-C0097,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD36-06-PHSEND000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-PHSEND-P0100,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-00-DCLENG000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGCON-MDCL37-20001,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-01-OPTAAD000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-OPTAAD-00218,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-02-ADCPTQ000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-ADCPTQ-01040,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-03-CTDBPC000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-CTDBPC-50111,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-04-DOSTAD000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-DOSTAD-00394,35.94988,-75.11943,,31,32,,,A
+AR82,,,,CP10CNSM-MFD37-07-ZPLSCC000,1,1,2024-04-03T13:57:00,,CGMCP-10CNSM-00001,,CGINS-ZPLSCC-55102,35.94988,-75.11943,,31,32,,,A
```
Comment (snwhite) [inline deployment/CP10CNSM_Deploy.csv:55]: Looks good.

File: deployment/CP11NOSM_Deploy.csv [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
+AR82,,,,CP11NOSM-MFD36-04-VELPTA000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-VELPTA-12711,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD36-05-PCO2WB000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-PCO2WB-C0096,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD36-06-PHSEND000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-PHSEND-P0069,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-00-DCLENG000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGCON-MDCL37-00007,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-01-OPTAAD000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-OPTAAD-00207,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-02-ADCPTF000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-ADCPTF-19221,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-03-CTDBPD000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-CTDBPD-50110,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-04-DOSTAD000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-DOSTAD-00428,36.17478,-74.82118,,106,107,,,A
+AR82,,,,CP11NOSM-MFD37-07-ZPLSCC000,1,1,2024-04-04T18:46:00,,CGMCP-11NOSM-00001,,CGINS-ZPLSCC-55101,36.17478,-74.82118,,106,107,,,A
```
Comment (snwhite) [inline deployment/CP11NOSM_Deploy.csv:45]: Looks good.

File: deployment/CP11SOSM_Deploy.csv [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+AR82,,,,CP11SOSM-MFD36-04-VELPTA000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-VELPTA-11417,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD36-05-PCO2WB000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-PCO2WB-C0098,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD36-06-PHSEND000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-PHSEND-P0158,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-00-DCLENG000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGCON-MDCL37-20003,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-01-OPTAAD000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-OPTAAD-00206,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-02-ADCPTF000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-ADCPTF-22672,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-03-CTDBPD000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-CTDBPD-50058,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-04-DOSTAD000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-DOSTAD-00130,35.72810,-74.85118,,104,105,,,A
+AR82,,,,CP11SOSM-MFD37-07-ZPLSCC000,1,1,2024-04-05T13:47:00,,CGMCP-11SOSM-00001,,CGINS-ZPLSCC-55103,35.72810,-74.85118,,104,105,,,A
```
Comment (snwhite) [inline deployment/CP11SOSM_Deploy.csv:45]: Looks good.

File: deployment/CP15MOAS-GL379_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR82,,,,CP15MOAS-GL379-00-ENG000000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,36.16192,-74.70431,,1000,1000,OM,,
+AR82,,,,CP15MOAS-GL379-01-ADCPAM000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49016,36.16192,-74.70431,,1000,1000,OM,,
+AR82,,,,CP15MOAS-GL379-02-FLORTM000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-08063,36.16192,-74.70431,,1000,1000,OM,,
+AR82,,,,CP15MOAS-GL379-03-CTDGVM000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09362,36.16192,-74.70431,,1000,1000,OM,,
+AR82,,,,CP15MOAS-GL379-04-DOSTAM000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00496,36.16192,-74.70431,,1000,1000,OM,,
+AR82,,,,CP15MOAS-GL379-05-PARADM000,1,1,2024-04-04T22:44:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50328,36.16192,-74.70431,,1000,1000,OM,,
```
Comment (snwhite) [inline deployment/CP15MOAS-GL379_Deploy.csv:7]: Looks good.

File: deployment/CP15MOAS-GL387_Deploy.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR82,,,,CP15MOAS-GL387-00-ENG000000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-01-ADCPAM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-02-FLORTM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04471,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-03-CTDGVM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09059,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-04-DOSTAM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00310,35.88224,-74.83170,,350,350,MA,,
+AR82,,,,CP15MOAS-GL387-05-PARADM000,1,1,2024-04-05T17:51:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50151,35.88224,-74.83170,,350,350,MA,,
```
Comment (snwhite) [inline deployment/CP15MOAS-GL387_Deploy.csv:7]: Looks good.  (I calculated 35.88223 from the QCT, but perhaps that was calculated from the ELog).

File: deployment/CP15MOAS-G0913_Deploy.csv [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+AR82,,,,CP15MOAS-G0913-00-ENG000000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGCON-GLDRCP-00913,35.88125,-74.83379,,200,200,CS,,
+AR82,,,,CP15MOAS-G0913-01-ADCPAM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-ADCPAM-73245,35.88125,-74.83379,,200,200,CS,,
+AR82,,,,CP15MOAS-G0913-02-FLORTM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-FLORTM-06642,35.88125,-74.83379,,200,200,CS,,
+AR82,,,,CP15MOAS-G0913-03-CTDGVM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-CTDGVM-09690,35.88125,-74.83379,,200,200,CS,,
+AR82,,,,CP15MOAS-G0913-04-DOSTAM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-DOSTAM-00909,35.88125,-74.83379,,200,200,CS,,
+AR82,,,,CP15MOAS-G0913-05-PARADM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-PARADM-50333,35.88125,-74.83379,,200,200,CS,,
```
Comment (snwhite) [inline deployment/CP15MOAS-G0913_Deploy.csv:7]: Looks good.

**Review by snwhite — APPROVED** [review]
All approved, but I had one question on a Lat value.

File: deployment/CP15MOAS-G0913_Deploy.csv (hunk 6 above)
Comment (cdobs) [inline deployment/CP15MOAS-G0913_Deploy.csv:7]: Looks good, but request we add AR82 for recover CUID

File: deployment/CP15MOAS-GL379_Deploy.csv (hunk 4 above)
Comment (cdobs) [inline deployment/CP15MOAS-GL379_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL387_Deploy.csv (hunk 5 above)
Comment (cdobs) [inline deployment/CP15MOAS-GL387_Deploy.csv:7]: looks good

**Review by cdobs — CHANGES_REQUESTED** [review]
Request one small change - add recover CUID to 913 deploy csv

File: deployment/CP15MOAS-GL387_Deploy.csv (hunk 5 above)
Comment (rtravis-whoi) [reply deployment/CP15MOAS-GL387_Deploy.csv:7]: yes I took it from the elog (the QLR table is calculated from that & rounded) - I threw a screenshot fr elog up in slack data channel.

File: deployment/CP15MOAS-G0913_Deploy.csv [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+AR82,,AR82,,CP15MOAS-G0913-00-ENG000000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGCON-GLDRCP-00913,35.88125,-74.83379,,200,200,CS,,
+AR82,,AR82,,CP15MOAS-G0913-01-ADCPAM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-ADCPAM-73245,35.88125,-74.83379,,200,200,CS,,
+AR82,,AR82,,CP15MOAS-G0913-02-FLORTM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-FLORTM-06642,35.88125,-74.83379,,200,200,CS,,
+AR82,,AR82,,CP15MOAS-G0913-03-CTDGVM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-CTDGVM-09690,35.88125,-74.83379,,200,200,CS,,
+AR82,,AR82,,CP15MOAS-G0913-04-DOSTAM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-DOSTAM-00909,35.88125,-74.83379,,200,200,CS,,
+AR82,,AR82,,CP15MOAS-G0913-05-PARADM000,1,1,2024-04-05T18:06:00,2024-04-05T21:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00913,CGINS-PARADM-50333,35.88125,-74.83379,,200,200,CS,,
```
Comment (cdobs) [inline deployment/CP15MOAS-G0913_Deploy.csv:7]: recover CUID looks good

**Review by cdobs — APPROVED** [review]

File: deployment/CP15MOAS-G0913_Deploy.csv (hunk 7 above)
Comment (snwhite) [reply deployment/CP15MOAS-G0913_Deploy.csv:7]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**